### ISSUE TITLE
Copying text on iOS could delete selected text

### ISSUE THREAD

**Issue by ocavue** [ProseMirror/prosemirror#1370]
## Steps to reproduce  

1. Open https://prosemirror.net/examples/basic/ using iOS Safari 
2. Select some words before links
3. Tap the "Copy" button in the tooltip
4. Switch to Notes app 
5. Switch back to Safari
6. The selected text is gone

## Video

https://share.cleanshot.com/k8pqxvLy

## Environment 

- iPhone 14
- iOS 16.4.1

**Comment by marijnh** [comment]
I think this is https://github.com/ProseMirror/prosemirror/issues/1135, which was found to be an iOS issue. Can you confirm that the same happens in a regular contentEditable element?

**Comment by ocavue** [comment]
Yes I can reproduce it with regular contentEditable element. Here is a reproduction video https://share.cleanshot.com/TBxtB0KB

**Comment by marijnh** [comment]
Right. So this is not a ProseMirror bug. I'm going to close this issue.
